.github/workflows/main.yml

```diff
@@ -33,14 +33,14 @@ jobs:
     if: github.ref != 'refs/heads/master'
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: Set up JDK 17
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v3
         with:
           java-version: 17
           distribution: 'temurin'
       - name: Set up cache
-        uses: actions/cache@v1
+        uses: actions/cache@v3
```

Comment:
Når du først er inne i workflows så kan vi gjerne fjerne bruken av actions/cache siden det er innebygget i setup-java, [se her for veilarb-eksempel](https://github.com/navikt/veilarboppfolging/blob/main/.github/workflows/deploy.yml#L26-L28) og [her for dokumentasjon](https://github.com/marketplace/actions/setup-java-jdk#caching-packages-dependencies)

Reply:
Det er slik det skal være? Vi har gjort dette i andre apper.